src/modules/authentication/auth-components/LoginForm.tsx

```diff
@@ -63,26 +54,27 @@ export const LoginForm: React.FC = () => {
 
   return (
     <div className='login-div'>
-      <form className='login-form'>
+      <form className='login-form' onSubmit={handleSubmit(onSubmit)}>
         <h1>Sign in</h1>
         <div className='input-login'>
           <label htmlFor='email'>Email</label>
           <input
             type='email'
-            name='email'
+            id='email-login'
             placeholder='Enter your email.'
-            onChange={handleEmailChange}
-            value={email}
-            required
+            {...register('email', { required: 'Email field is required.' })}
+            onChange={handleInputChange}
```

Comment:
ovime si overrideao ono što ti register daje, realno ne bi trebao tako hendlat auth error sad. Makni onchange